### DISCUSSION TITLE
Show preview of colour from hex codes

### DISCUSSION THREAD

**Discussion by bellebethcooper** [notaapp/talk#332] [Feature Requests] [answered]
Not sure if this is a good fit or too niche an idea, so feel free to close this request if so!

I recently found this feature in Bear, and I haven't found it in any other notes/Markdown app but it's super handy. If you type a colour hex code into bear like `#f7b100`, you'll get a little coloured circle to the left of that text, showing a preview of the colour. Similar to how some IDEs add a little colour preview when you're working in CSS.

The reason I find this so useful is that I work on a software project that uses lots of colours, which need to be consistent across several platforms. I use different IDEs when working on different code bases, so I need somewhere separate to the code to keep references to the colour palette, and I've had trouble finding anywhere to do this, which lets me name and group the colours as well as showing me a preview. In a Markdown file I can do whatever I want to group and sort the colours, but I need the preview for them to be useful, as I'm often looking through the colours to find "that one medium blue", or "what light grey options do we have?", etc.

Hope that makes sense!

**Comment by astoilkov** [comment, marked as answer]
We like this specific Bear feature a lot. We want to implement this. We still haven't only because it's a niche thing and we've been focusing on more priority stuff.

**Reply by bellebethcooper** [reply]
That makes sense!
